packages/calcite-components/src/components/button/button.scss

```diff
@@ -228,271 +253,288 @@
 :host([appearance]) {
   button,
   a {
-    @apply border-color-transparent
-      border
-      border-solid;
+    background-color: var(--calcite-button-background-color);
+    color: var(--calcite-button-text-color);
```

Comment:
These styles look like they should just be assigned with a generic `:host { button, a {}}` selector instead of here where it is looking just for the `appearance` attribute.  Adopting the pattern I described earlier will necessitate moving these styles to a generic `:host` selector anyway.